public/models/Interfaces.mock.ts

```diff
@@ -0,0 +1,120 @@
+import {
```

Comment:
Not feeling great about the naming for this file. Also I think we should separate the mocks for different types into separate files. I have accumulated all the types under the `types` folder at root level and will be removing this models folder altogether in a future PR soon

Reply:
The content is moved into different files and the file is removed.